src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.Ui;
+
+/**
+ * Command abstract class
+ */
+public abstract class Command {
+    protected String commandName;
+    protected String description;
+    protected String[] arguments;
+
+    public abstract void execute();
+
+    protected static LocalDate getDate(String date) throws DateTimeParseException {
```

Comment:
Not necessarily a Command-specific method. Could potentially be moved to a Utils class.

Reply:
I didn't have a use for a Utils class prior to making this function, so this was the most relevant place to put it. However, you raise a very good point here